src/main/java/org/galatea/starter/domain/IexHistoricalPrice.java

```diff
@@ -0,0 +1,20 @@
+package org.galatea.starter.domain;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.time.format.DateTimeFormatter;
```

Comment:
I don't see this import used anywhere. There are some ways in IntelliJ to manage imports for you - there's a checkbox in the Commit window to optimize imports on commit, and also an option in IntelliJ settings to optimize imports on the fly.

Reply:
Fixed when I was able to run checkstyle on the codebase.